File: sdk/python/examples/descriptors/nem_namespace.py
def descriptor_factory():
	return [
		# root namespace
		{
			'type': 'namespace_registration_transaction',
			'rental_fee_sink': 'TAMESPACEWH4MKFMBCVFERDPOOP4FK7MTDJEYP35',
			'rental_fee': 50000 * 1000000,
			'parent_name': '',
			'name': 'roger'
		},

		# child namespace
		{
			'type': 'namespace_registration_transaction',
			'rental_fee_sink': 'TAMESPACEWH4MKFMBCVFERDPOOP4FK7MTDJEYP35',
			'rental_fee': 1 * 1000000,
			'parent_name': 'roger',
			'name': 'charlie'
		}
	]

File: sdk/javascript/examples/descriptors/nem_namespace.js
export default () => ([
	// root namespace
	{
		type: 'namespace_registration_transaction_v1',
		rentalFeeSink: 'TAMESPACEWH4MKFMBCVFERDPOOP4FK7MTDJEYP35',
		rentalFee: 50000n * 1000000n,
		name: 'roger'
	},

	// child namespace
	{
		type: 'namespace_registration_transaction_v1',
		rentalFeeSink: 'TAMESPACEWH4MKFMBCVFERDPOOP4FK7MTDJEYP35',
		rentalFee: 1n * 1000000n,
		parentName: 'roger',
		name: 'charlie'
	}
]);
